src/common/QskRgbLiterals.h

```diff
@@ -0,0 +1,142 @@
+/******************************************************************************
+ * Copyright (C) 2023 Edelhirsch Software GmbH
+ * This file may be used under the terms of the 3-clause BSD License
+ *****************************************************************************/
+
+#ifndef QSK_RGB_LITERALS_H
+#define QSK_RGB_LITERALS_H
+
+namespace QskRgb
+{
+    namespace literals
+    {
+        /// @brief converts HTML #RRGGBB[AA] color hex string to Qt's RGB 0xAARRGGBB integer
+        /// @param str The string literal
+        /// @param len The length of the literal
+        /// @note non-hexadecimal characters are replaced with 0x0 value
+        constexpr unsigned int operator""_rgba( const char* const str, const size_t len )
```

Comment:
Should / could be Q_REQUIRED_RESULT + Q_DECL_CONSTEXPR

Reply:
Q_DECL_CONSTEXPR was a placeholder to make the code compilable without c++11.

Q_REQUIRED_RESULT is still valid - but when introducing these type of attributes it should be done throughout the complete qskinny code base. Maybe later ...

Reply:
Do we (or Qt) define any requirement towards a minimum C++ standard version?

Reply:
Qt6: C++17, since yesterday QSkinny has the same. So it is allowed to rely on C++17